test/acceptance/public-api-test.js

```diff
@@ -167,6 +167,67 @@ describe('public api', function() {
     });
   });
 
+  // TODO: extract this module in a file
+  // most of the tests will be here
+  // (incl. BOM)
+  describe('Linter.prototype.verifyAndFix', function() {
+    let basePath = null;
+    let linter;
+    let expected = {
+      rules: {
+        quotes: 'double',
+      },
+    };
+
+    beforeAll(async function() {
+      project = await createTempDir();
+      basePath = project.path();
+      project.write({
+        '.template-lintrc.js': `module.exports = ${JSON.stringify(expected)};`,
+        app: {
+          templates: {
+            'application.hbs': "<input class='mb4'>",
+          },
+        },
+      });
+    });
+
+    this.afterAll(async function() {
```

Comment:
Hmm, what is `this.afterAll`? I'm not familiar with that

Reply:
I copied it from above (test module for ˋverify` function). It might have to be fixed there too 🤔